wandi/mailer-bundle/1.0/config/packages/wandi_mailer.yaml

```diff
@@ -0,0 +1,4 @@
+wandi_mailer:
+    sender:
+        name: 'example'
```

Comment:
You can also use the env variable for define the name & address like this PR : https://github.com/symfony/recipes-contrib/pull/398/files and then explain the next step in the post-install

Reply:
@maxhelias thanks for the advice